MatterControlLib/PrinterCommunication/Drivers/X3G/X3GReader.cs

```diff
@@ -34,27 +36,20 @@ public string translate(byte[] x3gResponse, string relatedGCommand, out bool com
 		private class X3GPacketAnalyzer
 		{
 			private byte[] response;
+			private PrinterConfig printer;
 			private X3GCrc crc;
 			private string gCommandForResponse; //Figure out better name. this is the gCommand that was sent to the printer that caused this response
 			private X3GPrinterDetails printerDetails; //used to get location information and other needed response data
 			private StringBuilder temperatureResponseStrBuilder; //Saves extruder temp when we have a heated bed to send back temps together
 
-			public X3GPacketAnalyzer(X3GPrinterDetails PtrDetails)
+			public X3GPacketAnalyzer(X3GPrinterDetails PtrDetails, PrinterConfig printer)
 			{
+				this.printer = printer;
 				crc = new X3GCrc();
 				printerDetails = PtrDetails;
 				temperatureResponseStrBuilder = new StringBuilder();
 			}
 
-			public X3GPacketAnalyzer(byte[] x3gResponse, string relatedGCommand, X3GPrinterDetails PtrDetails)
-			{
-				response = x3gResponse;
-				crc = new X3GCrc();
-				gCommandForResponse = relatedGCommand;
-				printerDetails = PtrDetails;
-				temperatureResponseStrBuilder = new StringBuilder();
-			}
-
```

Comment:
@mmoening  - I removed this constructor as it wasn't used, needed to be updated with the new printer param and figured we'd restore if you thought we needed it. Can you review when you get a chance?